Controller/HelperController.php

```diff
@@ -291,6 +291,35 @@ public function setObjectFieldValueAction(Request $request)
             $value = filter_var($value, FILTER_VALIDATE_BOOLEAN);
         }
 
+        // Handle entity choice association type, transforming the value into entity
+        if ('' !== $value && $fieldDescription->getType() == 'choice' && $fieldDescription->getOption('class')) {
+
+            // Get existing associations for current object
+            $associations = $admin->getModelManager()
+                ->getEntityManager($admin->getClass())->getClassMetadata($admin->getClass())
+                ->getAssociationNames();
+
+            if (!in_array($field, $associations)) {
+                return new JsonResponse(array(
+                    'status' => 'KO',
+                    'message' => sprintf('Unknown association "%s", association does not exist in entity "%s", available associations are "%s".',
+                        $field, $this->admin->getClass(), implode(', ', $associations)),
+                ));
+            }
+
+            $value = $admin->getConfigurationPool()->getContainer()->get('doctrine')->getManager()
+                ->getRepository($fieldDescription->getOption('class'))
+                ->find($value);
+
+            if (!$value) {
+                return new JsonResponse(array(
+                    'status' => 'KO',
```

Comment:
@greg0ire Statuscode 400 for all KO responses in method setObjectFieldValueAction(), right? Or only for my patched part?